src/test-utilities/legacy.tsx

```diff
@@ -174,40 +160,13 @@ export function polarisContextReactWrapper<P, S>(
   element: React.ReactElement<P>,
   {app}: AppContextOptions,
 ): PolarisContextReactWrapper<P, S> {
-  function TestProvider<P>(props: P) {
-    let content: React.ReactNode = element;
-
-    if (Object.keys(props).length > 0) {
-      content = React.cloneElement(React.Children.only(element), props);
-    }
-
-    return (
-      <I18nContext.Provider value={app.intl}>
-        <ScrollLockManagerContext.Provider value={app.scrollLockManager}>
-          <StickyManagerContext.Provider value={app.stickyManager}>
-            <ThemeProviderContext.Provider value={app.themeProvider}>
-              <AppBridgeContext.Provider value={app.appBridge as any}>
-                <LinkContext.Provider value={app.link}>
-                  <FrameContext.Provider value={app.frame}>
-                    {content}
-                  </FrameContext.Provider>
-                </LinkContext.Provider>
-              </AppBridgeContext.Provider>
-            </ThemeProviderContext.Provider>
-          </StickyManagerContext.Provider>
-        </ScrollLockManagerContext.Provider>
-      </I18nContext.Provider>
-    );
-  }
+  const appBridge: any = app.appBridge;
```

Comment:
I also just spotted that the appBridge typing in tests is kinda wonky. I'll try and come up with a plan once this and #1828 are merged.

Reply:
Yup app bridge is using `{}` in tests as a mock but isn't actually a valid type 😬